### DISCUSSION TITLE
dynamic env-files

### DISCUSSION THREAD

**Discussion by RonStein2000** [go-task/task#433] [Ideas]
I would find it helpful to have the ability to add vars in the path of the dotenv files, e.g. 
  dotenv: ["docker/local/{{.User}}/some.config"]
to have local setups for individual users/machines.

with user defined like this:

vars:
    sh: echo {{if eq OS "windows"}}"$USERNAME"{{else}}"$(whoami)"{{end}}

**Comment by mgbowman** [comment]
See #434

**Comment by andreynering** [comment]
Hi @RonStein2000,

This was just implemented at 08265ed1d7833bf3bbc4d0f060f7e3277e6f1d3e.
